src/grappler/ngraph_optimizer.cc

```diff
@@ -66,18 +66,81 @@ Status NgraphOptimizer::Optimize(tensorflow::grappler::Cluster* cluster,
   if (config::IsEnabled() == false ||
       std::getenv("NGRAPH_TF_DISABLE") != nullptr) {
     NGRAPH_VLOG(0) << "Ngraph is disabled ";
+    graph.ToGraphDef(output);
```

Comment:
should we move this to the very top? the if
```
if (config::IsEnabled() == false ||
      std::getenv("NGRAPH_TF_DISABLE") != nullptr)
```

Reply:
Keeping it here allows us to dump graphs even though ngraph is disabled. Helps in scoping new models.

Reply:
Yes.